src/components/Brand.vue

```diff
@@ -23,5 +21,9 @@ export default {
     font-variant: common-ligatures small-caps;
     margin-left: -2px;
   }
+  a {
+    color: #000 !important;
```

Comment:
Overriding colors like this will affect theme(dark , light themes). Colors should always come from UI library.